README.md

```diff
@@ -272,7 +272,7 @@ As you can see, the class `GreeterImpl` implements the interface
       responseObserver.onCompleted();
     }
 ```
-- `hello` takes two parameters:
+- `sayHello` takes two parameters:
```

Comment:
I can change it, but the method being referred to above is really `sayHello`, isn't it?

```Java
public void sayHello(HelloRequest req, StreamObserver<HelloReply> responseObserver) {
```

It also says this just above:

> ... [IDL](https://github.com/grpc/grpc-java/tree/master/examples/src/main/proto) by implementing the method `sayHello`:

Should I still change it?